api/rs/slint/mcu.md

```diff
@@ -379,17 +373,17 @@ loop {
 
 ```
 
-Hint: In our experience, using the synchronous `DrawTarget::fill_contiguous` function is slow. If
+Note: In our experience, using the synchronous `DrawTarget::fill_contiguous` function is slow. If
 your device is capable of using DMA, you may be able to achieve better performance by using
 two line buffers: One buffer to render into with the CPU, while the other buffer is transferred to
 the screen using DMA asynchronously.
 
-Hint: If you see wrong colors on your device, it may be necessary to invert the bits in the u16 pixel
+Note: If you see wrong colors on your screen, it may be necessary to invert the bits in the u16 pixel
 representation before sending them to the screen driver.
 
```

Comment:
```suggestion
```

If the user uses DrawTarget::fill_contiguous, this shouldn't be a problem at all.
otherwise it might just be a misconfigured driver or something. I believe most developer would figure that out by themselves.

Reply:
Ok, if you think so. We can add it again if it becomes a FAQ.